src/guide/reviewing-and-sponsoring-a-change.md

```diff
@@ -0,0 +1,73 @@
+# Reviewing and Sponsoring a Change
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [GitHub Docs on pull requests reviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews)
+:::
+
+Two of the most important contributions you can make in the OpenJDK community are to review and sponsor changes for other developers. All changes needs to be reviewed. Many changes even need to be reviewed by more than one person. This means that in order to enable a fast-pased development experience, all developers need to review more changes then they produce themself.
+
+If you are new to an area, reviewing changes is also a great way to learn the code and see what general styles and types of changes that are relevant in the area. Be mindful though, if you don't know the area well you should state this in your review comments. As a **R**eviewer you have a responsibility to make sure changes are sound and goes in line with the general direction of the area. If you, as a **R**eviever, review a change in an area that you don't know well you probably shouldn't be the one to approve the change.
+
+New developers in the OpenJDK community don't have the permissions needed to push changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get the first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually push the sponsored contribution into the repository.
+
+Sponsoring new contributions is an important activity - it's how the engineering culture of a project gets passed on from the core group to new Contributors, from one generation to the next. It should be fun, so please celebrate the contributions you've sponsored by mentioning them on your blog or similar. Congratulate other Sponsors on their work. Take pride in the value you provide to Contributors. Their success reflects well on you.
+
+As a Sponsor, Contributors will look up to you for guidance to get their contributions into the project - your actions will determine whether Contributors will feel welcome and want to engage further with the project beyond their initial attempt, or not. Let's not lose enthusiastic, engaged and technically competent Contributors due to a lack of communication. If there is a request in your area of expertise and you can't address it, at least acknowledge receipt of the request and provide an estimate for when you'll be able to give it your attention. A frank explanation of your time constraints or commitments will be appreciated and respected.
```

Comment:
This, to me, makes being a Sponsor sound like an active role i.e. you sit and watch all day for things that your might Sponsor; or there is a notion of a designated sponsor for an area who may not have time for something at the moment. But normally it is not like that. If a change has been socialized ahead of time then a sponsor tends to emerge from the participants in the PR. If a change has not been socialized and a PR just appears then it may well just sit there because it isn't on anyone's radar - not even to say "I'm too busy for this" because they are too busy to pay it any attention in the first place.